Jenkinsfile.nightly

```diff
@@ -14,25 +21,29 @@ pipeline {
 				'''
 			}
 		}
-		stage('Run integration tests') {
+		stage('Run integration tests without syncing') {
 			steps {
-				sh 'npm test -- mocha:extensive:integration'
+				sh 'MAXIMUM_TRANSACTION=1000 SYNCING_DISABLED=true npm test -- mocha:extensive:integration'
 			}
 		}
-		stage('Run integration tests w/o broadcasting') {
+		stage('Run integration tests without broadcasting') {
 			steps {
-				sh 'BROADCASTING_DISABLED=true npm test -- mocha:extensive:integration'
+				sh 'MAXIMUM_TRANSACTION=1000 BROADCASTING_DISABLED=true npm test -- mocha:extensive:integration'
 			}
 		}
-		stage('Run integration tests w/o syncing') {
+		stage('Run integration tests with syncing and broadcasting') {
 			steps {
-				sh 'SYNCING_DISABLED=true npm test -- mocha:extensive:integration'
+				sh 'MAXIMUM_TRANSACTION=1000 npm test -- mocha:extensive:integration'
 			}
 		}
 	}
 	post {
 		always {
 			sh './node_modules/.bin/pm2 kill'
+			archiveArtifacts artifacts: "logs/*", allowEmptyArchive: true
+			dir('logs/') {
+				deleteDir()
+			}
```

Comment:
Add `cleanWs()` it will clean everything. And that's what we use in other Jenkin files.